app/src/App.js

```diff
@@ -134,4 +139,32 @@ const Form = () => {
   );
 };
 
+//formik
+
+const SignupForm = () => {
+  const formik = useFormik({
+    initialValues: { email: "" },
+    onSubmit: (values) => {
+      alert(JSON.stringify(values, null, 2));
+    },
+  });
+
+  return (
+    <>
+      <h1>Formik Form</h1>
+      <form onSubmit={formik.handleSubmit}>
+        <label htmlFor="email">Email Address</label>
+        <input
+          id="email"
+          name="email"
+          type="email"
+          onChange={formik.handleChange}
+          value={formik.values.email}
+        />
+        <button type="submit">Submit</button>
```

Comment:
If the email input is clicked in and then out without entering an email, the “Required” div that shows up pushes the button down to the next line. It’s better to just have the button down to the next line to start.
I suggest adding a `<br>` in between here as a quick fix.